projects/plugins/beta/src/admin/admin.js

```diff
@@ -217,6 +233,10 @@
 	 * @param {object} element - DOM Element object.
 	 */
 	function hide( element ) {
+		if ( ! element ) {
```

Comment:
Where is this getting called from without an element?

Reply:
Thinking this was added before I returned out of one of the other functions, but not necessary anymore after https://github.com/Automattic/jetpack/pull/20390/commits/cc6cbf64e8d989853b07025af8c9e4930efc41c0